Formula/visp.rb

```diff
@@ -48,14 +40,17 @@ def install
                          "-DUSE_JPEG=ON",
                          "-DJPEG_INCLUDE_DIR=#{Formula["jpeg"].opt_include}",
                          "-DJPEG_LIBRARY=#{Formula["jpeg"].opt_lib}/libjpeg.dylib",
-                         "-DUSE_LAPACK=OFF",
+                         "-DUSE_LAPACK=ON",
+                         "-DLAPACK_INCLUDE_DIR=#{Formula["lapack"].opt_include}",
+                         "-DLAPACK_LIBRARY=#{Formula["lapack"].opt_lib}/liblapack.dylib",
                          "-DUSE_LIBUSB_1=OFF",
                          "-DUSE_OPENCV=ON",
                          "-DOpenCV_DIR=#{Formula["opencv"].opt_share}/OpenCV",
                          "-DUSE_PCL=OFF",
                          "-DUSE_PNG=ON",
                          "-DPNG_PNG_INCLUDE_DIR=#{Formula["libpng"].opt_include}",
                          "-DPNG_LIBRARY_RELEASE=#{Formula["libpng"].opt_lib}/libpng.dylib",
+                         "-DUSE_PCL=OFF",
```

Comment:
This is already above, so there is clearly something weird here

Reply:
Should be ok now